src/coreComponents/codingUtilities/CMakeLists.txt

```diff
@@ -30,6 +31,9 @@ blt_add_library( NAME       codingUtilities
                  OBJECT     ${GEOSX_BUILD_OBJ_LIBS}
                )
 
+# Avoid compiling with nvcc which sometimes crashes on fast_float
+set_source_files_properties( Parsing.cpp PROPERTIES LANGUAGE CXX )
```

Comment:
what is the standard `LANGUAGE` property value when running on a machine with GPU?

Reply:
BLT automatically sets `LANGUAGE=CUDA` for all source files in any target that has a dependency on `cuda` target